File: Models/Question.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EducationalSoftwareServer.Models
{
    public class Question
    {
        public int QuestionId { get; set; }
        public int? TestId { get; set; }
        public string Description { get; set; }
        public string Type { get; set; }
        public string Category { get; set; }
        public List<QuestionAnswer> QuestionAnswers { get; set; }

    }
}

File: Client/EducationalSoftwareClient/src/app/models/question.ts
import { QuestionAnswer } from "./questionAnswer";

export class Question {
    questionId: number;
    testId: number;
    description: string;
    type: string;
    category: string;
    questionAnswers: QuestionAnswer[];

    constructor(){
        this.questionId = 0;
        this.testId = 0;
        this.description = "";
        this.type = "";
        this.category = "";
        this.questionAnswers=[];
    }
}
